Always use the Git SHA for deployments

emileswarts opened the pull request:
This was optional but given we will always be deploying through CircleCI,
there is no need to guard against it not being present.

emileswarts committed:
Always use the Git SHA for deployments

This was optional but given we will always be deploying through CircleCI,
there is no need to guard against it not being present.

asmega commented:
runner currently isn't going through circleci and the images are being generated by hand. what is the benefit of this over leaving backwards compatibility?

emileswarts commented:
Runner only builds platform images, and doesn't use Helm to deploy.  So that script won't be touched luckily.
Backwards compatibility will still be preserved, and all the apps that deploy using this deploy_platforms script will be run through CI.

emileswarts committed:
Git SHA is a required argument

Not failing on this will result in an empty value being passed through to Helm.
Fail the deployment instead if this argument is not supplied.
